plugins/catalog-react/src/components/EntityRefLink/EntityRefLink.tsx

```diff
@@ -25,16 +25,26 @@ import { Link, LinkProps } from '@backstage/core-components';
 import { useRouteRef } from '@backstage/core-plugin-api';
 import { Tooltip } from '@material-ui/core';
 
+type SubRoutePath = `${'#' | '/'}${string}`;
+
 export type EntityRefLinkProps = {
   entityRef: Entity | EntityName;
   defaultKind?: string;
   title?: string;
   children?: React.ReactNode;
+  subRoutePath?: SubRoutePath;
```

Comment:
Hmm, should this be a [subRouteRef](https://backstage.io/docs/plugins/composability#subroutes)? 🤔

Reply:
I don't think so. It's a path (trailing static part) on top of a ref that gets resolved dynamically, right? Possibly could be named `subPath` though.

Reply:
We'll change it to be `subPath`